roles/postgres_upgrade/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+
+# This view for Datadog references system tables, which blows up if we try to upgrade the cluster
+# with it included. We can safely drop it here and re-add it later via the datadog role.
+- name: drop pg_stat_activity_dd view # noqa 301
+  command: "psql {{ db }} -c 'DROP VIEW IF EXISTS pg_stat_activity_dd CASCADE;'"
```

Comment:
There was a pretty nasty issue related to this view, which wasn't caught by the compatibility checks, and meant the upgrade failed half-way through, leaving both clusters non-operational. :scream: